src/ResourceManagement/HDInsight/HDInsight/Properties/AssemblyInfo.cs

```diff
@@ -32,8 +32,8 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.1.2.0")]
+[assembly: AssemblyVersion("1.2.0.0")]
```

Comment:
This should remain at 1.0.0.0, only the file version should match the package version.  This makes it easier for libraries to take a forward-compatible depoendency on your assembly